blogs/2022/10/04/vscode-community-discussions.md

```diff
@@ -0,0 +1,61 @@
+---
+Order: 77
+TOCTitle: VS Code Community Discussions
+PageTitle: VS Code Community Discussions for Extension Authors
+MetaDescription: Announcing the official launch of VS Code Community Discussions, a place for extension authors to connect.
+Date: 2022-10-04
+Author: Olivia Guzzardo
+---
+
+# VS Code Community Discussions for Extension Authors
+
+October 4, 2022 by Olivia Guzzardo, [@OliviaGuzzardo](https://twitter.com/OliviaGuzzardo)
+
+The true power of VS Code comes from its vast extension ecosystem, which only exists because of our incredible community of extension authors. Whether creating a new language extension to make it possible to program in virtually any language, developing a new theme to help with productivity, or extending the workbench to help with a unique developer workflow, extension authors help millions of people by sharing their creation.
+
+Our goal is, and has always been, to make extension development accessible to everyone. And while we have plenty of [documentation](https://code.visualstudio.com/api) for getting started with extension development, sometimes there are questions that documentation just can’t answer. Sometimes, all you need is a friendly hand from someone who has been there, had that question, and figured it out.
+
+Today we're announcing the official launch of [VS Code Community Discussions](https://github.com/microsoft/vscode-discussions/discussions): a place for extension authors to ask questions, connect with the community, and showcase their work.
+
+## Navigating the Forum
+
+![A screenshot showing the VS Code Community Discussions landing page](github-discussions.png)
+
+The new GitHub Discussions forum is a place for extension authors to connect with each other and stay up to date on announcements from the VS Code team. We want to be as close to developers as possible, and having a centralized, official community helps to achieve that.
+
+When you visit the Discussions repo, you'll notice that there are currently 3 categories...
+
+### Announcements
+
+This is where you can keep up with all of the latest information for extension authors from the VS Code team. This includes things like recently released features such as "Pre-release extensions" as well as some giveaways we have from time-to-time just for our extension author communities - like [free Azure Credits](https://github.com/microsoft/vscode-discussions/discussions/135)!
+
+### Extension Development Q&A
```

Comment:
Should this be "Extension Development QnA" to match the site?
Or is it too late to change the Discussion site to be "Q & A"? 
"QnA" looks odd to me

Reply:
And you've already taken the screenshot :-)